presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -66,12 +66,25 @@ Mathematical Functions
 
     Returns the value of ``string`` interpreted as a base-``radix`` number.
 
+.. function:: hypergeometric_cdf(populationSize, numberOfSuccesses, sampleSize, observedSuccesses) -> double
```

Comment:
Please change `observedSuccesses` to `value` (just to be consistent with the other functions)

(also in the function itself)